airflow-core/tests/unit/api_fastapi/common/test_parameters.py

```diff
@@ -18,11 +18,72 @@
 from __future__ import annotations
 
 import re
+from typing import Annotated
 
 import pytest
-from fastapi import HTTPException
+from fastapi import Depends, FastAPI, HTTPException
 
-from airflow.api_fastapi.common.parameters import SortParam
+from airflow.api_fastapi.common.parameters import FilterParam, SortParam, filter_param_factory
+from airflow.models import DagRun, Log
+
+
+class TestFilterParam:
+    def test_filter_param_factory_description(self):
+        app = FastAPI()  # Create a FastAPI app to test OpenAPI generation
+        expected_descriptions = {
+            "dag_id": "Filter by DAG ID Description",
```

Comment:
```suggestion
            "dag_id": "Filter by Dag ID Description",
```

Whehter to use Dag or dag is still under discussion, but definitely not DAG is confirmed

Reply:
Thanks for the catch that's my typo.